src/Core/Field.h

```diff
@@ -345,7 +346,9 @@ class Field
     }
 
     template <typename T>
-    Field(T && rhs, enable_if_not_field_or_stringlike_t<T> = nullptr);
+    Field(T && rhs, enable_if_not_field_or_bool_or_stringlike_t<T> = nullptr);
+
+    Field(bool rhs);
```

Comment:
I'd prefer to implement this constructor right here because it's just a one-liner.